### DISCUSSION TITLE
A problem with the NPM support page

### DISCUSSION THREAD

**Discussion by so1ve** [community/community#22503] [New to GitHub]
<p>Hello,<br>
When I send a request for help on <code>https://www.npmjs.com/support</code>, it prompts me “error loading page”.<br>
I looked at the network page and found that <code>POST /support</code> returned a 403, and a JSON parsing error. Instead of JSON, it returned an HTML page. This is the DDoS Defense page of cloudflare.<br>
How can I deal with it so that this situation will not happen again?</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/1/510592f8e4041a7d8494cdd8e81f15f9309e6033.png" data-download-href="/uploads/short-url/byKChZdggPGKunxhXdWrxT1akGn.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071518-b615384a-323e-4580-91dd-25830a5feb54.png" alt="image" data-base62-sha1="byKChZdggPGKunxhXdWrxT1akGn" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/1/510592f8e4041a7d8494cdd8e81f15f9309e6033_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/5/1/510592f8e4041a7d8494cdd8e81f15f9309e6033_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/1/510592f8e4041a7d8494cdd8e81f15f9309e6033_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/1/510592f8e4041a7d8494cdd8e81f15f9309e6033_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 154 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>seems no error on my end at this time,</p>

**Comment by so1ve** [comment]
<p>Hello there,<br>
the problem did not disappear.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/7/37b85b9ae7c1706387a9aabc370bdca01b5c30e9.png" data-download-href="/uploads/short-url/7WVfevkLdwqzHAN19HgvnaTtHrH.png?dl=1" title="" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/58381667/181071519-d4f3d96c-447f-4d82-af83-ed637222f813.png" alt="" data-base62-sha1="7WVfevkLdwqzHAN19HgvnaTtHrH" width="690" height="329" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37b85b9ae7c1706387a9aabc370bdca01b5c30e9_2_690x329.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37b85b9ae7c1706387a9aabc370bdca01b5c30e9_2_1035x493.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37b85b9ae7c1706387a9aabc370bdca01b5c30e9_2_1380x658.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/7/37b85b9ae7c1706387a9aabc370bdca01b5c30e9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename"></span><span class="informations">1920×917 112 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>your JavaScript is turned off ? try turning it on</p>

**Comment by jfullstackdev** [comment]
<p>this is another one, Microsoft Edge, still no error</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/8/b812b3d0ba2917b4594eccaad4bc1f7217447ded.png" data-download-href="/uploads/short-url/qgnViVfP54lY7twecProa9aqaLP.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181071522-036944ac-8906-48c7-9959-824446391902.png" alt="image" data-base62-sha1="qgnViVfP54lY7twecProa9aqaLP" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b812b3d0ba2917b4594eccaad4bc1f7217447ded_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b812b3d0ba2917b4594eccaad4bc1f7217447ded_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b812b3d0ba2917b4594eccaad4bc1f7217447ded_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/8/b812b3d0ba2917b4594eccaad4bc1f7217447ded_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 91.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by so1ve** [comment]
<p>I think I’ve turned it on… I’m using the default settings of the Edge browser, where JavaScript is already turned on.</p>

**Comment by so1ve** [comment]
<p>Yes, very strange. My friends don’t have this problem either.</p>

**Comment by jfullstackdev** [comment]
<p>how about firewall ? antivirus ? website scanner ?</p>
<p>sometimes even ISP</p>

**Comment by so1ve** [comment]
<p>It doesn’t change after I clear the cookie.<br>
I only opened Windows Defender. There are no other plugins in the browser</p>

**Comment by so1ve** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="8" data-topic="230539">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>ISP</p>
</blockquote>
</aside>
<p>Maybe it’s the ISP problem… I’m in China</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>how about using VPN,  give it a try</p>
<aside class="quote no-group" data-username="so1ve" data-post="7" data-topic="230539">
<div class="title">
<div class="quote-controls"></div>
 so1ve:</div>
<blockquote>
<p>My friends don’t have this problem either.</p>
</blockquote>
</aside>
<p>your friends are located the same as yours ? mainland China ?</p>

**Comment by so1ve** [comment]
<p>Oops<br>
When I use a VPN, this problem no longer occurs</p>

**Comment by so1ve** [comment]
<p>Thank you very much for your help<br>
When I use VPN, this problem no longer appears</p>
